### DISCUSSION TITLE
[Feature] Mobile search - find based on image existing/uploading status

### DISCUSSION THREAD

**Discussion by kerhbal** [immich-app/immich#8363] [Feature Request]
### The feature

First of all, this is such an amazing project!
The feature I want to bring up is to find images that do not exist on current device. This info is already presented in the app, like the cloud mark without check mark in it. I just want to list all these photos, and ability to select/delete them.
The reason is I will from time to time delete some photos from iOS Photos, since who knows how many garbage/blurry meaningless photos were added during a trip when I don't have time to clean them up lol.
And if immich can show all the photos do not exist on the device, I'll have the ability to remove them from immich as well.
Thanks!

------
Edited:
I think this can be more generic: image existing/uploading status.
There are 3 types of that:
- uploaded
- exists on mobile device, but not uploaded
  - this will be helpful to find the "ghost" photos, somehow I have some of those: mobile app tells me all photos have been uploaded, but some will show the mark saying they are not uploaded. currently there is no way to find them at once.
- exists in server, but not on this current mobile device
  - this is the scenario mentioned above

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Nenormalniy** [comment]
Yeah, that filter is defititely needed.

**Comment by bo0tzz** [comment]
#2155
